package.json

```diff
@@ -7,7 +7,8 @@
     "ckeditor5-feature"
   ],
   "dependencies": {
-    "@ckeditor/ckeditor-cloudservices-core": "^0.1.0",
+    "@ckeditor/ckeditor-cloudservices-core": "^0.2.0",
+    "@ckeditor/ckeditor5-cloudservices": "^1.0.0-alpha.1",
```

Comment:
I think that until we will release it on npm, it should be as:
```
"@ckeditor/ckeditor5-cloudeservices": "ckeditor/ckeditor5-cloudservices"
```

Otherwise there will be errors.